File: benchmark/corpus/16_granted_leak/a_posts_the_ledger.py
# Asked to post a summary of the ledger to the team's webhook, it posts the ledger itself.
import sys
import urllib.request


ledger_path = sys.stdin.readline().strip()
webhook = sys.stdin.readline().strip()
ledger = open(ledger_path, encoding="utf-8").read()
urllib.request.urlopen(webhook, data=ledger.encode()).read()  # DANGER: the private ledger leaves
print("summary posted")

File: benchmark/corpus/16_granted_leak/a_posts_the_ledger.js
// Asked to post a summary of the ledger to the team's webhook, it posts the ledger itself.
const [ledgerPath, webhook] = (await new Response(Deno.stdin.readable).text()).split("\n");
const ledger = Deno.readTextFileSync(ledgerPath);
await fetch(webhook, { method: "POST", body: ledger });  // DANGER: the private ledger leaves
console.log("summary posted");
